lapce-core/Cargo.toml

```diff
@@ -33,6 +33,7 @@ tree-sitter = "0.20.10"
 tree-sitter-bash = { git = "https://github.com/tree-sitter/tree-sitter-bash", rev = "4488aa41406547e478636a4fcfd24f5bbc3f2f74", optional = true }
 tree-sitter-c = { version = "0.20.6", optional = true }
 tree-sitter-cpp = { version = "0.20.0", optional = true }
+tree-sitter-d = { version = "0.3.8", optional = true }
```

Comment:
I believe you need the actual tree-sitter grammar for D.  I was literally following the pattern used elsewhere.

Note that the crate associated with tree-sitter-d was only created by me last night.  But it is a crate now.

Reply:
Um. Ok.  (Btw, if this was documented somewhere it would have helped me.)

What do I need to do to make lapce find my grammar if not this?  (Note that this is a *different* grammar than the one gishadow added -- the grammar he used is older and has some flaws. I created my grammar about a year ago because of defects in the cybershadow grammar.)

Reply:
Also, the queries have to match the grammar!  My queries are written against my grammar, and won't work with the other one.

Reply:
I'll try updating the submodule instead, now that I know about it. :-)